pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/participant/OsteoParticipantService.java

```diff
@@ -117,20 +117,33 @@ private static void createOsteoTag(ElasticSearchParticipantDto participantDto, S
                                        DDPInstance osteoInstance, String tagName) {
         // invariant: participant should have a DSM entity
         Dsm dsm = participantDto.getDsm().orElseThrow();
-        createOteoTag(dsm, ddpParticipantId, osteoInstance, tagName);
+        createOsteoTag(dsm, ddpParticipantId, osteoInstance, tagName);
```

Comment:
Just saw this, we can remove the unused inputs to this method later

Reply:
Not sure I follow.
Can you explain?